test/unit/client.test.js

```diff
@@ -41,19 +41,19 @@ const MOCK_CONVERSATION = {
   ]
 };
 
-test('constructor | throws if options are not provided', t => {
+test.serial('constructor | throws if options are not provided', t => {
```

Comment:
instead of `test.serial` on every test, you can just add `-s` to the ava cmd in package.json.

Reply:
I like that it's explicit so if I run them without package.json I know it's required.